infrastructure/cdn-in-a-box/bin/ats-version.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+trap 'echo "Error on line ${LINENO} of ${0}" >/dev/stderr; exit 1' ERR;
+set -o errexit -o nounset -o pipefail
+
+project=trafficserver
+script_dir="$(dirname "$0")"
+ats_version_file="${script_dir}/../cache/ATS_VERSION"
+
+remote_ats_version() {
+  local gitbox_url=https://gitbox.apache.org/repos/asf
+  local repo="${project}.git"
+	local branch refs commit last_tag release
```

Comment:
Inconsistent indentation. You can use whatever you want as long as it's consistent, but just remember that tabs are inherently superior and if you use spaces you're a bad person.

Reply:
Using good-person spacing in a2a18f830b